packages/noports_core/test/sshnp/util/sshnp_initial_tunnel_handler/sshnp_initial_tunnel_handler_test.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:noports_core/sshnp_foundation.dart';
+import 'package:test/test.dart';
+import 'package:mocktail/mocktail.dart';
+
+class StubbedSshnpInitialTunnelHandler extends Mock
+    with SshnpInitialTunnelHandler<String> {}
+
+void main() {
+  group('SshnpInitialTunnelHandler', () {
+    late final StubbedSshnpInitialTunnelHandler handler;
+    setUp(() {
+      handler = StubbedSshnpInitialTunnelHandler();
+    });
+    test('public API', () async {
+      when(() => handler.startInitialTunnel(identifier: 'asdf'))
+          .thenAnswer((invocation) async => 'Called');
+
+      await expectLater(
+        await handler.startInitialTunnel(identifier: 'asdf'),
+        'Called',
+      );
```

Comment:
Testing this is blocked by some other bits, will revisit the rest of this in a later PR.

Reply:
Wrong item.